scripts/python-skywater-pdk/skywater_pdk/generate_symbolator_plots.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
```

Comment:
I think this should be named something more like `generate_symbols.py`

Reply:
fixed